Data-Structures-and-Algos-Topic/Module1-Intro-to-Data-Structures-and-Algos/Activity2_HashTables/checkpoints/9-Checkpoint.md

```diff
@@ -0,0 +1,12 @@
+# name
+Run time of "Length of the longest substring without repeating characters"
+
+# cards_folder
+curriculum/Data-Structures-and-Algos-Topic/Module1-Intro-to-Data-Structures-and-Algos/Activity2_HashTables/checkpoints/
+
+# instruction
```

Comment:
- punctuation. it can be a period.
- you can say: "find and explain the runtime of...."